### PR TITLE
docs: re-update assets for specs

### PR DESCRIPTION
**Pull request by seolaoh**
Re-updated some assets to properly represent Kanvas' architecture.

### CODE REVIEW

**Comment by chokobole** [comment]
Could you tell me why you changed the assets?

If you intend to use abstract terms in the overview, it would be good to have consistent terminology throughout the document, as some parts refer to 'Execution Engine' while others use 'Execution Engine(EE).'

Is it correct that you juste removed spaces in the last picture?

File: specs/assets/architecture.svg
Comment (seolaoh) [inline specs/assets/architecture.svg:1]: unify component terms to Batcher, Rollup Node, Validator, Execution Engine

File: specs/assets/components.svg
Comment (seolaoh) [inline specs/assets/components.svg:1]: add spacing between arrow lines

File: specs/assets/propagation.svg
Comment (seolaoh) [inline specs/assets/propagation.svg:1]: unify component terms to Execution Engine, KanvasPortal (not Portal)

File: specs/assets/user-withdrawing-to-l1.svg
Comment (seolaoh) [inline specs/assets/user-withdrawing-to-l1.svg:1]: just remove spaces in L2OutputOracle and BatchInBox

**Comment by seolaoh** [comment]
> Could you tell me why you changed the assets?
> 
> If you intend to use abstract terms in the overview, it would be good to have consistent terminology throughout the document, as some parts refer to 'Execution Engine' while others use 'Execution Engine(EE).'
> 
> Is it correct that you juste removed spaces in the last picture?

I added some comments to each picture.

> Could you tell me why you changed the assets?
> 
> If you intend to use abstract terms in the overview, it would be good to have consistent terminology throughout the document, as some parts refer to 'Execution Engine' while others use 'Execution Engine(EE).'
> 
> Is it correct that you juste removed spaces in the last picture?

@chokobole I added some comments to each picture.

**Review by chokobole — APPROVED** [review]
lgtm

**Pull request merged**